apps/omg_utils/lib/omg_utils/http_rpc/client_adapter.ex

```diff
@@ -56,12 +58,44 @@ defmodule OMG.Watcher.HttpRPC.Adapter do
 
   def get_response_body(error), do: {:error, {:client_error, error}}
 
+  @doc """
+  Decodes specified keys in map from hex to binary
+  """
+  @spec decode16(map(), list()) :: map()
```

Comment:
where is the requirement for this coming from?

Reply:
We need to deep-decode hex values from the json responses that the RPC clients are giving us. Also note that this code is being moved.